### PR TITLE
Feat(CX-3381): Redesign swa Landing page P6 [Testimonials, Footer]

### PR DESCRIPTION
**Pull request by lordkiz**
This PR resolves [CX-3381] 

### Description

https://user-images.githubusercontent.com/18648835/222420967-9e0b6d20-46d1-4bcb-868a-fe207c3cb559.mov

https://user-images.githubusercontent.com/18648835/222420997-e398c0c1-c1fd-410e-964a-217644d2861e.mov

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Redesign swa Landing page P6 [Testimonials, Footer] - kizito

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3381]: https://artsyproduct.atlassian.net/browse/CX-3381?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lordkiz** [comment]
I might need to purge my `main`. Since the last release I get references to past commits when i rebase with main.

File: src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+  return (
+    <Flex mt={1}>
+      <FlatList
+        data={REVIEWS}
+        horizontal
+        showsHorizontalScrollIndicator={false}
+        keyExtractor={(item) => item.reviewText}
+        pagingEnabled
+        viewabilityConfigCallbackPairs={viewabilityConfigCallbackPairs.current}
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx:74]: Nice

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx [hunk 2]
```diff
@@ -120,27 +122,38 @@ export const SellWithArtsyHome: React.FC<SellWithArtsyHomeProps> = ({
             {enableNewSWALandingPage && <Spacer y={4} />}
             <SellWithArtsyRecentlySold recentlySoldArtworks={recentlySoldArtworks!} />
 
-            <Spacer y={4} />
-
-            <WhySellWithArtsy />
+            {enableNewSWALandingPage && (
+              <Join separator={<Spacer y={4} />}>
+                <></>
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:127]: 👀

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx [hunk 3]
```diff
@@ -120,27 +122,38 @@ export const SellWithArtsyHome: React.FC<SellWithArtsyHomeProps> = ({
...
+                    <Text variant="sm">Submit an Artwork</Text>
+                  </Button>
+                </Flex>
+                <Spacer y={4} />
+              </>
+            )}
 
-            <Footer />
+            <Footer onConsignPress={handleConsignPress} />
+            {enableNewSWALandingPage && <Spacer y={4} />}
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:156]: Is this not needed for the old landing page?

**Review by olerichter00 — APPROVED** [review]
Great work ⭐

**Review by dariakoko — APPROVED** [review]
LGTM

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx (hunk 3 above)
Comment (lordkiz) [reply src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:156]: In the old landing page there was no Spacer after Footer.

Comment (olerichter00) [reply src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:156]: 👍

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx (hunk 2 above)
Comment (olerichter00) [reply src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:127]: 👍

**Pull request merged**